app/pb_kits/playbook/pb_phone/docs/_phone_default.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= pb_rails("phone", props: {
+  icon: "mobile-android",
+  number: "3491859988"
```

Comment:
I tried passing a string that might come through nitro `349-185-9988` & it breaks.

We want to first strip any non-numeric characters. Then after the number is only numbers, use this to format the number `number_to_phone(your_formatted_number, area_code: true)`

Reply:
You could use:
`"(627)222-2777".delete('^0-9')`
or 
`"(627)222-2777".tr('^0-9', '')`

Reply:
Tried with different formats and it worked fine